spec/unit/pdk/validate/puppet_syntax_spec.rb

```diff
@@ -36,7 +40,7 @@
       let(:options) { { auto_correct: true } }
 
       it 'has no effect' do
-        expect(command_args).to eq(%w[parser validate].concat(targets))
+        expect(command_args).to eq(%w[parser validate --config /dev/null].concat(targets))
```

Comment:
Is the purpose of this existing `auto_correct` test assertion to make sure that some option is not being added? If so should we maybe just assert that the option is absent rather than having to update this test every time the invocation changes?

Also should we have a new test that explicitly checks for the `--config` param?